src/v/storage/tests/storage_test_fixture.h

```diff
@@ -128,12 +128,12 @@ class storage_test_fixture {
           .get0();
     }
 
-    // clang-format off
     template<typename T = random_batches_generator>
-        requires requires (T generator) {
-            { generator() } -> std::same_as<ss::circular_buffer<model::record_batch>>;
-        }
-    // clang-format on
+    requires requires(T generator) {
+        {
+            generator()
+            } -> std::same_as<ss::circular_buffer<model::record_batch>>;
```

Comment:
this does not quite nice. but this is what i have using clang-format-14 with existing rules.